deploy/backplane/mkrts/redhat-kas-fleetshard/10-mkrts-fleetshard-admins.SubjectPermission.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mkrts-admins
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: admin
+    namespacesAllowedRegex: (^redhat-managed-kafka-.*|^redhat-kas-fleetshard-.*)
```

Comment:
```suggestion
    namespacesAllowedRegex: (^redhat-managed-kafka-.*|^redhat-kas-fleetshard-.*|^managed-application-services-observability)
```

Reply:
@david-martin Could you clarify what managed-application-services-observability ns does, why it's not name with prefix "redhat-"?

Reply:
The managed-application-services-observability namespace is where the observability stack that monitors the managed-kafka components is deployed.

Reply:
> why it's not name with prefix "redhat-"?

@wanghaoran1988 I can't recall the reason for that name being used. 
Is it a problem do you think?
I know there are some references to `redhat-` in other places, similar to `openshift-` that maybe mean we should change the prefix? (dedicated-admin/reader role i think?)

@pb82 do you recall the reason for the name?

Reply:
I can't recall a particular reason. The observabilit stack can be installed in any namespace, but as you said, there might be other namespace filters that expect a `redhat-` prefix.

Reply:
@david-martin I am asking because the ns used by addon service on osd all follow the convention to start with "redhat-", if this is just an exception, I am ok with that.